app/views/idv/finance/new.html.slim

```diff
@@ -0,0 +1,15 @@
+- title t('idv.titles.welcome')
+
+.mt1.mb3.h6.caps.red = t('idv.titles.welcome')
+h2.heading = t('idv.titles.session.finance')
+p.italic Please provide only one of the following:
+= simple_form_for(idv_finance_form, url: idv_finance_path,
+    html: { autocomplete: 'off', method: 'put', role: 'form' }) do |f|
+  = f.error_notification
+  = f.collection_radio_buttons :finance_type, Idv::FinanceForm.finance_type_choices,
+    :first, :last, required: true, item_wrapper_tag: :div
```

Comment:
Also, it looks like we lost the styling we used to have, and the fact that each radio button had its own input field. I think we want separate input fields because the CCN field has to be exactly 8 digits, so I think we want to preserve the maxlength and pattern attributes we used to have.